10.0/Dockerfile

```diff
@@ -9,7 +9,18 @@ RUN apt-get update && apt-get install -y pwgen && rm -rf /var/lib/apt/lists/*
 
 RUN mkdir /docker-entrypoint-initdb.d
 
-RUN apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys 199369E5404BD5FC7D2FE43BCBCB082A1BB943DB
+RUN apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys 199369E5404BD5FC7D2FE43BCBCB082A1BB943DB \
+	&& apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys 430BDF5C56E7C94E848EE60C1C4CBDCDCD2EFD2A
+
+RUN echo "deb http://repo.percona.com/apt jessie main" > /etc/apt/sources.list.d/percona.list \
+	&& { \
+		echo 'Package: *'; \
+		echo 'Pin: release o=Percona Development Team'; \
+		echo 'Pin-Priority: 999'; \
```

Comment:
With this at the same priority as the MariaDB repo below, there's a chance (even a minor one) that some other dependency of MariaDB gets pulled from this repo accidentally -- I think it would be prudent to change this priority to be something slightly smaller (so it's still preferred over the standard Debian repos, but less than the MariaDB repos), like 998 (since the current release has a priority of 990, which is why MariaDB's at 999; https://manpages.debian.org/cgi-bin/man.cgi?query=apt_preferences).